src/js/common/schemaform/review/ObjectField.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import {
+  getDefaultFormState,
+  orderProperties,
+  retrieveSchema,
+  shouldRender,
+  getDefaultRegistry,
+} from 'react-jsonschema-form/lib/utils';
+
+/*
+ * This is largely copied from the react-jsonschema-form library,
+ * but with the way descriptions are used changed
+ */
+
+class ObjectField extends React.Component {
+  static defaultProps = {
+    uiSchema: {},
+    errorSchema: {},
+    idSchema: {},
+    registry: getDefaultRegistry(),
+    required: false,
+    disabled: false,
+    readonly: false,
```

Comment:
Yeah, I need to pass those to `SchemaField` or remove them. This component is a copied and modified version of one in the `react-jsonschema-form` library, so we don't necessarily use all of the props that could be passed to it.